packages/atlas/src/components/_auth/SignInModal/SignInSteps/SignInModalWalletStep.tsx

```diff
@@ -35,27 +43,34 @@ export const SignInModalWalletStep: FC<SignInStepProps> = ({
 
   const wallets = useMemo(() => {
     const unsortedWallets = getWalletsList()
-    const hasPolkaWallet = unsortedWallets.some((wallet) => wallet.extensionName === POLKAWALLET)
     if (isMobileDevice) {
-      if (hasPolkaWallet) {
-        return unsortedWallets
-          .filter((wallet) => wallet.extensionName === POLKAWALLET)
-          .map((wallet) => ({
-            ...wallet,
-            title: capitalizeFirstLetter(wallet.title),
-            installed: wallet.installed,
-            logo: { src: polkaWalletLogo, alt: 'Polkawallet logo' },
-          }))
-      }
-      return [
-        {
-          title: 'Polkawallet',
-          extensionName: POLKAWALLET,
+      const allMoblieWallets = new Set([
+        'polkawallet',
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ...Object.keys((window as any).injectedWeb3 || {}),
+        ...unsortedWallets
+          .filter((wallet) => wallet.extensionName !== 'talisman')
```

Comment:
Why are we filtering Talisman here?

Reply:
Talisman doesn't have mobile wallet, but it would be shown as a Recommended choice. All because of using @talisman-connect/wallets to get "supported wallets", this list is only adjusted for the standard web user.